platform/kubernetes/release.go

```diff
@@ -114,8 +114,15 @@ func deploymentExec(def *apiext.Deployment, newDef *apiObject) regradeExecFunc {
 				"deployment", newDef.Metadata.Name,
 				"--namespace", newDef.Metadata.Namespace,
 			)
-			logger.Log("cmd", strings.Join(cmd.Args, " "))
 			err = cmd.Run()
+			logger.Log(
```

Comment:
Could this be arranged like the other snippet above: assemble `args`, use them to construct `cmd` and also log them?